lambeq/backend/drawing/drawing.py

```diff
@@ -458,7 +501,9 @@ def _get_box_color(box: grammar.Diagrammable,
             color = FRAME_COLORS[(frame_attr - 1) % len(FRAME_COLORS)]
 
     return color
-
+def _get_wire_color(wire_id):
```

Comment:
This can be moved to `drawing_backend.py`, I think. The same is true for `_get_box_color`.